### ISSUE TITLE
Since version 0.30, disabling LineLength doesn't work.

### ISSUE THREAD

**Issue by dmarcoux** [sds/haml-lint#303]
Hello,

First of all, thank you for `haml-lint`, it's great!

We want to update `haml-lint` to the version `0.30`, but somehow, our continuous integration is now failing due to a disable comment not working. It worked before, so this is surprising to us.

It happens in this [PR](https://github.com/openSUSE/open-build-service/pull/7527) and the continuous integration is [here](https://circleci.com/gh/openSUSE/open-build-service/24572). The offense happens in this [file](https://github.com/openSUSE/open-build-service/blob/19d4872280a149e69314d18e5ab72d2f000654a6/src/api/app/views/webui2/webui/staging/excluded_requests/index.html.haml#L7-L9).

Thank you!

**Comment by Ana06** [comment]
The concrete bug is the following code, which produces an offense:

```ruby
-# haml-lint:disable LineLength
%table.responsive.table.table-sm.table-striped.table-bordered.w-100#excluded-requests-datatable{ data: excluded_requests_remote_url(@staging_workflow.id) }

**Comment by sds** [comment]
Thanks for the report. Would really appreciate it if someone could take the time to perform a [bisect](https://git-scm.com/docs/git-bisect) to find the problematic commit!

**Comment by rob-murray** [comment]
Hi @sds 

I'm also having this issue - it seems to be this commit: 5b4ebf93749ad091553847c18fe7239d64c62a83

I haven't had a chance to have a look at the code, just did a quick test using the files [here](https://gist.github.com/rob-murray/416bd01b2a9a1244fd39f3b4b489b6f3)

Using the commit prior to 5b4ebf93749ad091553847c18fe7239d64c62a83 works as expected

**Comment by sds** [comment]
Thanks for finding the problematic commit. Tagging @michaelherold as they may have some context on what could have broken.

I'm open to reverting the commit if we can't find a resolution, but want to give folks a chance to fix first. Hoping someone can step up to help. Thanks!

**Comment by michaelherold** [comment]
Thanks, @Ana06, for the reproduction. I'll take a look when I get the chance, but don't feel like you should wait for me.

**Comment by michaelherold** [comment]
I pushed a commit into a pull request in #307 that takes your example line as a test case. It's not failing. I'm wondering what else is causing the problem?

Could you expand on that test case to see if we can track down a minimal reproduction script? Feel free to commit directly to that branch.

**Comment by shaneog** [comment]
I'm also experiencing this issue. Below is the problematic line.
```haml
-# haml-lint:disable LineLength
= javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js', integrity: 'sha384-rY/jv8mMhqDabXSo+UCggqKtdmBfd3qC2/KvyTDNQ6PcUJXaxK1tMepoQda4g5vB', crossorigin: 'anonymous'
```
